### ISSUE TITLE
Cannot persist after updating to 1.0.0

### ISSUE THREAD

**Issue by hjw0968** [Seb-L/pinia-plugin-persist#28]
electron V17
Vue3.2.29
vite2.8.6
typescript4.6.2

<img width="414" alt="image" src="https://user-images.githubusercontent.com/22829164/158000895-acddf393-b0b6-40a1-b841-d7594d2070da.png">
<img width="865" alt="image" src="https://user-images.githubusercontent.com/22829164/158000908-bbc50542-9c8d-492c-af10-aef79a789a30.png">

version 0.0.7 is normal

**Comment by Nakus0426** [comment]
Yes, I have the same problem.When I change the state, there is nothing in the storage

**Comment by rationalthinker1** [comment]
I have this problem too. Had to downgrade to 0.0.93.

**Comment by kagaricyan** [comment]
I found the difference: at 0.0.93, use vueDemi.watch to trigger update.
